scio-bigquery/src/main/scala/com/spotify/scio/bigquery/package.scala

```diff
@@ -176,10 +176,9 @@ package object bigquery {
 
   /** Utility for BigQuery `TIMESTAMP` type. */
   object Timestamp {
-
-    // FIXME: verify that these match BigQuery specification
-    // YYYY-[M]M-[D]D[ [H]H:[M]M:[S]S[.DDDDDD]][time zone]
+    // YYYY-[M]M-[D]D[( |T)[H]H:[M]M:[S]S[.DDDDDD]][time zone]
```

Comment:
So there could be either a space or `T` between date and hour? It's not captured in the pattern though?

Reply:
Yeah, either one is valid
With existing code, when we save to BQ tables it always saves with `T` but when reads it will work with either one.

Reply:
Ah gotcha. This is the formatter so we only handle one case. The parser below handles both with 2 `appendOptional`.